bindings/c/Cargo.toml

```diff
@@ -13,6 +13,9 @@ cbindgen = "0.24.0"
 [dependencies]
 bytes = "1.5.0"
 lazy_static = "1.4.0"
-libsql = { path = "../../libsql", features = ["encryption"] }
```

Comment:
This is not an acceptable change. Most of our projects that use C bindings depend on the encryption feature.